tests/Collector/MigrationsFlattenerTest.php

```diff
@@ -20,8 +20,7 @@
 
 class MigrationsFlattenerTest extends TestCase
 {
-    /** @var MigrationsFlattener */
-    private $flattener;
+    private MigrationsFlattener $flattener;
```

Comment:
Argh, looks like I forgot to remove this: https://github.com/doctrine/DoctrineMigrationsBundle/blob/b9d29ea70ca2e4a7abf90f92775526440aff4122/phpcs.xml.dist#L24-L29